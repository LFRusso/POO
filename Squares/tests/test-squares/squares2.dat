XYZ 0.01 2.1 0.5
UVW -0.4 1.3 1.0
