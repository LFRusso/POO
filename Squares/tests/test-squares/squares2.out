XYZ intercepts UVW with area 0.1
